pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -71,8 +71,17 @@ type chartValues struct {
 	Global struct {
 		Images struct {
 			IstioPilot struct {
+				Name string `json:"name"`
 				Version string `json:"version"`
-			} `json:"istio_pilot"`
+				Directory string `json:"directory"`
+				ContainerRegistryPath string `json:"containerRegistryPath"`
+			} `json:"istio_pilot"`;
+			IstioProxyV2 struct {
+				Name string `json:"name"`
+				Version string `json:"version"`
+				Directory string `json:"directory"`
+				ContainerRegistryPath string `json:"containerRegistryPath"`
+			} `json:"istio_proxyv2"`
```

Comment:
Do we need all these properties? Are they all used? I only included the mandatory one not to pass stuff pointlessly.